src/transformState.js

```diff
@@ -5,7 +5,31 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const { type } = action;
+
+    switch (type) {
+      case 'addProperties':
+        (Object.assign(state, action['extraData']));
```

Comment:
```suggestion
        (Object.assign(state, action.extraData));
```
There is no need to use dynamic keys. Consider using static keys or using destructuring